openapi-maven-plugin/src/main/java/io/github/kbuntrock/configuration/parser/CommonParserUtils.java

```diff
@@ -0,0 +1,73 @@
+package io.github.kbuntrock.configuration.parser;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.github.kbuntrock.MojoRuntimeException;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.project.MavenProject;
+
+/**
+ * @author Kévin Buntrock
+ */
+public final class CommonParserUtils {
+
+	private CommonParserUtils() {
+	}
+
+	/**
+	 * Get the content from an undetermined input (file path or directly the content)
+	 *
+	 * @param mavenProject
+	 * @param input        a path or directly the content
+	 * @return the content
+	 */
+	public static String getContentFromFileOrText(final MavenProject mavenProject, final String input) {
+		if(StringUtils.isEmpty(input)) {
+			return null;
+		}
+
+		String content = input;
+
+		// We can load a file if the attribute represent a path
+		final String url = mavenProject.getBasedir() + FileSystems.getDefault().getSeparator() + input;
+		Path path = null;
+		try {
+			path = Paths.get(url);
+		} catch(final Exception ex) {
+			// non parseable path. Probably not a path
```

Comment:
At least log it in warn - future developers will be grateful
```suggestion
			logger.warn("Cannot read URL {} to Path", url, ex);
```

Reply:
I'm counting on the exception launched a bit after if nothing can be parsed to give a good clue on what's going wrong. 
I'll not change this for the moment, but I keep your remark in mind if I have feedbacks on this functionality ones it will be released.